### PR TITLE
Add nestif linter

### PR DESCRIPTION
**Pull request by nakabonne**
Fixes https://github.com/golangci/golangci-lint/issues/941

### CODE REVIEW

File: pkg/golinters/nestif.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			}
+			var issues []nestif.Issue
+			for _, f := range pass.Files {
+				issues = append(issues, checker.Check(f, pass.Fset)...)
+			}
+			if len(issues) == 0 {
+				return nil, nil
+			}
+
+			sort.Slice(issues, func(i, j int) bool {
```
Comment (ernado) [inline pkg/golinters/nestif.go:43]: Please use [SliceStable](https://golang.org/pkg/sort/#SliceStable)

File: vendor/modules.txt [hunk 2]
```diff
@@ -122,6 +122,8 @@ github.com/mitchellh/go-homedir
 github.com/mitchellh/go-ps
 # github.com/mitchellh/mapstructure v1.1.2
 github.com/mitchellh/mapstructure
+# github.com/nakabonne/nestif v0.3.0
```
Comment (ernado) [inline vendor/modules.txt:125]: Please rebase and remove vendoring. So sorry to ask it.

**Comment by ernado** [comment]
Thank you! LGTM with two minor points to resolve.

**Comment by nakabonne** [comment]
@ernado Hi, thank you for your confirmation. I've modified the part that was pointed out, could you confirm that?

**Pull request merged**